### DISCUSSION TITLE
label_values - how to query the value?

### DISCUSSION THREAD

**Discussion by RichardWright** [grafana/grafana#44934] [Dashboards, Panels & Visualizations]
I have this query - label_values(some_metric_name{kubernetes_namespace="$namespace", test="This"} >0,theLabelIWant)

But no matter how I chop it, I can't filter values?

Is this even possible?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
